NOTICE

```diff
@@ -17,9 +17,42 @@ under the auspices of the Eigenbase Foundation
 and released as the LucidDB project.
 
 
+================= Apache Flink =================
```

Comment:
https://github.com/apache/incubator-druid/pull/7388 was raised on March 22 and it contains the source code of the master branch. It was middle of the [1.8 release process](https://github.com/apache/flink/releases), so the version would be 1.8.1.